test/types/queries.test.ts

```diff
@@ -231,7 +231,7 @@ async function gh11041(): Promise<void> {
   // 3. Create a Model.
   const MyModel = model<User>('User', schema);
 
-  expectType<HydratedDocument<User> | null>(await MyModel.findOne({}).populate('someField').exec());
+  expectType<Omit<HydratedDocument<User>, never> | null>(await MyModel.findOne({}).populate('someField').exec());
```

Comment:
I am not very happy about this change, but I didnt figure out why I have to do this. But the type is basically the same. 

@mohammad0-0ahmad
Any thoughts?

Reply:
@Uzlopak  I've mentioned u in another PR couldn't push to this.

Reply:
Closing in favor of yours